Packs/Core/Playbooks/playbook-Impossible_Traveler.yml

```diff
@@ -367,7 +367,7 @@ tasks:
           left:
             value:
               complex:
-                root: inputs.UserManagerEmail
+                root: UserManagerEmail
```

Comment:
From where will we get the userManagerEmail data?

Reply:
There is a sub-playbook called "Impossible Traveler - Enrichment", and inside him there is a sub-playbook called "Active Directory - Get User Manager Details" that outputs the Manager email to "UserManagerEmail"